frontend/utilities/helpers.tsx

```diff
@@ -843,8 +843,13 @@ export const getUniqueColumnNamesFromRows = (rows: any[]) =>
     )
   );
 
+export const addCommasToCount = (count: number) => {
+  return count.toString().replace(/\B(?=(\d{3})+(?!\d))/g, ",");
```

Comment:
I think you can just use `count.toLocaleString()` [instead](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString#using_tolocalestring)

Reply:
nice!